dev-packages/e2e-tests/test-applications/nuxt-3/pages/client-error.vue

```diff
@@ -1,11 +1,16 @@
 <script setup>
 import ErrorButton from '../components/ErrorButton.vue';
+
+const catchErr = () => {
+  console.log('Additional functionality in NuxtErrorBoundary');
+}
 </script>
 
 <template>
-  <ErrorButton id="errorBtn"  error-text="Error thrown from Nuxt-3 E2E test app"/>
-  <ErrorButton id="errorBtn2" error-text="Another Error thrown from Nuxt-3 E2E test app"/>
-</template>
-
-
+  <ErrorButton id="errorBtn"  error-text="Error thrown from Nuxt-4 E2E test app"/>
+  <ErrorButton id="errorBtn2" error-text="Another Error thrown from Nuxt-4 E2E test app"/>
 
+  <NuxtErrorBoundary @error="catchErr">
+    <ErrorButton id="errorBtn3" error-text="Error thrown in Error Boundary"/>
```

Comment:
l: we could have a more expressive button id here, would make reading the tests a bit easier too.